02 Fx/03 Angular 2/README.md

```diff
@@ -5,19 +5,19 @@ In this sample we are going to setup a basic Angular 2 app with Webpack.
 We will start from sample _02 Fx/00 TypeScript_.
 
 Summary steps:
- - Installing Angular 1.x libraries.
- - Creating the app.
- - Instantiating the app from the html.
- - Creating a component (inline HTML).
- - Creating a service.
- - Displaying the component.
- - Creating an external HTML template and consuming it.
+ - Install Angular 1.x libraries.
```

Comment:
If we change `Summary steps` I think we have to change [root Readme](https://github.com/Lemoncode/webpack-2.x-by-sample/blob/master/README.md)